_includes/sections/our-professions/development.md

```diff
@@ -150,3 +150,22 @@
    communicating about our work allows clients to see how we’re doing, and what
    needs their attention, and gives everyone in the team context for the work
    being done and decisions being made.
+
+#### Career progression for developers
+
+We have a
+[public progression framework](https://dxw.progressionapp.com/technology)
+describing the skills expected of each role across the technology team. Your
+progression doesn’t depend on having done everything in the framework for the
+new role but on demonstrating that you could do them if asked.
+
+We track changes to the framework in the
```

Comment:
This sentence feels a little long - I think it could be broken into two:
> We track changes to the framework in the changelog, where we describe what changed and _why_ we changed it. We keep track of any future changes we may want to make or think about in the backlog.

Reply:
Good shout. Thanks.